db_service/db_handler.go

```diff
@@ -0,0 +1,15 @@
+package db_service
+
+import (
+	"context"
+	"github.com/daniyakubov/book_service_n/models"
+)
+
+type DBHandler interface {
```

Comment:
Please [read](https://go.dev/doc/effective_go#interface-names) the convention in go for interfaces.
furthermore - IMO this interface indicates the functionality of your book store, so maybe rename0ng it to `BookStorer` will indicates more the essence of the interface